src/api/groups.js

```diff
@@ -0,0 +1,67 @@
+import express from 'express';
+
+import { configureUsersService } from '../config.js';
+import { groupSchema, validateBody } from './validation.js';
+
+const groupsRouter = express.Router();
+const usersService = configureUsersService();
+
+groupsRouter.route('/')
+    .post(validateBody(groupSchema), async (req, res, next) => {
+        const group = req.body;
```

Comment:
I would add everything into try block. Chance this line of code throws exception is low - but never zero. Somebody could add additional extraction, mapping which potentially can lead to exception (for example, cannot read property of undefined) and it will crash the app

Reply:
added all lines to try block